test/notification-system.test.js

```diff
@@ -56,12 +56,12 @@ describe('Notification Component', function() {
 
   it('should be rendered', done => {
     component = TestUtils.findRenderedDOMComponentWithClass(instance, 'notifications-wrapper');
-    expect(component).toExist();
+    expect(component).not.toBeNull();
```

Comment:
maybe I'm wrong, but as I understand it was:
```component !== void 0``` and you made ```component !== null``` and this is completely different checks ?